velox/connectors/hive/HiveConnector.h

```diff
@@ -202,13 +202,22 @@ class HiveDataSource : public DataSource {
 class HiveConfig {
  public:
   /// Can new data be inserted into existing partitions or existing
-  /// unpartitioned tables
+  /// unpartitioned tables.
   static constexpr const char* FOLLY_NONNULL kImmutablePartitions =
       "hive.immutable-partitions";
 
+  /// Maximum number of partitions per writer.
+  static constexpr const char* FOLLY_NONNULL kMaxPartitionsPerWriters =
```

Comment:
Let's documents this configuration property in https://facebookincubator.github.io/velox/configs.html